nixos/doc/manual/administration/containers.chapter.md

```diff
@@ -0,0 +1,28 @@
+# Container Management {#ch-containers}
+
+NixOS allows you to easily run other NixOS instances as *containers*.
+Containers are a light-weight approach to virtualisation that runs
+software in the container at the same speed as in the host system. NixOS
+containers share the Nix store of the host, making container creation
+very efficient.
+
+::: {.warning}
```

Comment:
Is this valid markdown syntax?

Reply:
I don't think it's valid commonmark: It's a pandoc extension called `fenced_divs`.
In this case it should be equivalent to `<div class="warning">`.
I count at least 30 occurences in `doc`, not sure how this will work after we switch away from pandoc.

Reply:
We better highlight the content using this for now. I think it is possible to change this as this is not too hard work, any suggestions? For example, shall I change all

```
::: {.warning}
foo foo foo
:::
```

to

```
**Warning:** foo foo foo
```

?

Reply:
We are currently fine with this extension.